pkg/scalers/influxdb_scaler.go

```diff
@@ -47,7 +61,29 @@ func NewInfluxDBScaler(config *scalersconfig.ScalerConfig) (Scaler, error) {
 		return nil, fmt.Errorf("error parsing influxdb metadata: %w", err)
 	}
 
-	logger.Info("starting up influxdb client")
+	if meta.influxVersion == "3" {
+		logger.Info("starting up influxdb v3 client")
+
+		clientv3, err := influxdb3.New(influxdb3.ClientConfig{
+			Host:     meta.serverURL,
+			Token:    meta.authToken,
+			Database: meta.database,
+		})
+
+		if err != nil {
+			panic(err)
```

Comment:
we should not panic here

Reply:
updated, please let me know if it looks ok